influxdb/client.py

```diff
@@ -6,14 +6,14 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import itertools
```

Comment:
Rather than importing the whole `itertools` module, how about just importing what's needed (`islice` & `chain`)?